client/packages/inventory/src/Stocktake/DetailView/ContentArea/columns.ts

```diff
@@ -239,25 +201,7 @@ export const useStocktakeColumns = ({
         key: 'difference',
         label: 'label.difference',
         align: ColumnAlign.Right,
-        getSortValue: row => {
-          if ('lines' in row) {
-            const { lines } = row;
-            const total =
-              lines.reduce(
-                (total, line) =>
-                  total +
-                  (line.snapshotNumberOfPacks -
-                    (line.countedNumberOfPacks ?? line.snapshotNumberOfPacks)),
-                0
-              ) ?? 0;
-            return (total < 0 ? Math.abs(total) : -total).toString();
-          } else {
-            return (
-              row.snapshotNumberOfPacks -
-                (row.countedNumberOfPacks ?? row.snapshotNumberOfPacks) ?? ''
-            );
-          }
-        },
+        sortable: false,
```

Comment:
Sorting by difference also doesn't seem too important? Although could be handy to see which items appear to have gone missing?